eth/chains/mainnet/__init__.py

```diff
@@ -39,7 +40,10 @@ def validate_header(cls, header, previous_header, check_seal=True):
         super().validate_header(header, previous_header, check_seal)  # type: ignore
 
         # The special extra_data is set on the ten headers starting at the fork
-        extra_data_block_nums = range(cls.dao_fork_block_number, cls.dao_fork_block_number + 10)
+        extra_data_block_nums = range(
+            cls.get_dao_fork_block_number(),
+            cls.get_dao_fork_block_number() + 10,
```

Comment:
Now that this had to split lines, might be worth getting the fork number first, so the range can be on one line. Like:

```py
fork_num = cls.get_dao_fork_block_number()
extra_data_block_nums = range(fork_num, fork_num + 10)
```